src/main/java/com/userservice/Entity/Users.java

```diff
@@ -0,0 +1,21 @@
+package com.userservice.Entity;
+
+import com.userservice.enums.CountryCode;
+import jakarta.persistence.*;
+import lombok.*;
+@Entity
+@Getter
+@Setter
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Users extends BaseEntity{
```

Comment:
`@NoArgsConstructor` is not required as logically no one is going to initialize the user without any arguments. Please remove this.